lib/subordinateio.c

```diff
@@ -232,7 +232,7 @@ static const struct subordinate_range *find_range(struct commonio_db *db,
                 return NULL;
         }
         owner_uid = pwd->pw_uid;
-        sprintf(owner_uid_string, "%lu", (unsigned long int)owner_uid);
+        snprintf(owner_uid_string, sizeof (owner_uid_string), "%lu", (unsigned long int)owner_uid);
```

Comment:
If you're going to do this, it would be nice if you would check that the return value is not <0 and not >= sizeof(owner_uid_string).

Reply:
included check